EZP-32239: Logging and cache revision (#1309)
ezsystems/developer-documentation: docs/guide/cache/shop_caching.md

|`SilversolutionsEshopBundle:Navigation:showMenu`|Left menu|Tag: `siso_menu`|
|`SilversolutionsEshopBundle:Navigation:showMenu`|Main menu|Tag: `siso_menu`|

All cache blocks are described in the `silver_eshop.default.http_cache` parameter in `parameters.yml`:

``` yaml
silver_eshop.default.http_cache:
    product:
        # Response max age in seconds. Zero means that this response will not be cached.
        max_age: 28800
        vary: ~
    product_list:
        max_age: 28800
        vary: ~
```

## Shop cache tags

[[= product_name_com =]] uses cache tags to tag and purge content.

|Tag|Used for|Purged|
|--- |--- |--- |
|`siso_basket_<id>`|Basket preview in the header|By event, on basket change|
|`siso_menu`|Main menu and left side menus (product catalog)||
|`content-<contentid>`|Textmodules|When content (text modules) are changed|
|`siso_user_<userid>`|User-specific data (shows name of the user)|When the user logs in or out|

## Purging caches

> You can purge shop HTTP cache using the `ibexa:commerce:purge-http-cache` command.

Reviewer: ```suggestion
You can purge the shop's HTTP cache with the `ibexa:commerce:purge-http-cache` command.
```

> By default the command purges all shop-related HTTP cache.

Reviewer: ```suggestion
By default, the command purges all shop-related HTTP caches.
```
?

> You can specify cache identifiers to purge as space-separated list, for example:

Reviewer: ```suggestion
You can specify cache identifiers to purge as a space-separated list, for example:
```
